docker-compose.yml

```diff
@@ -1,17 +1,20 @@
 version: '3.2'
 services:
+
   db:
     image: postgres:11.4
     ports:
       - "127.0.0.1:5432:5432"
     environment:
       - POSTGRES_USER=${USER}
   redis:
-    image: redis:latest
+    image: redis:5.0
     ports:
       - "127.0.0.1:6379:6379"
   mailcatcher:
-    image: tophfr/mailcatcher
+    build:
```

Comment:
@mbonica1990 There are some people who will not be using the Docker installation. Won't this break for them?